cinema/serializers.py

```diff
@@ -106,11 +120,10 @@
 
 
 class MovieSessionDetailSerializer(MovieSessionSerializer):
-    movie = MovieListSerializer(many=False, read_only=True)
+    movie = MovieDetailSerializer(many=False, read_only=True)
```

Comment:
In `MovieSessionDetailSerializer`, the `movie` field is using `MovieDetailSerializer`, which includes all fields of the movie. Ensure that this level of detail is necessary for the use case, as it might include more data than needed.